src/commands/generateProject.ts

```diff
@@ -56,9 +54,15 @@ export async function generateProject(clickedFileUri: vscode.Uri | undefined): P
         url: yamlInputURL!, // eslint-disable-line @typescript-eslint/no-non-null-assertion
         method: "GET",
       };
-      const downloadLocation = path.join(tmpDirPath, "openapi.yaml");
-      await fileUtil.downloadFile(requestOptions, downloadLocation);
-      yamlInputFileURI = vscode.Uri.parse(downloadLocation);
+      try {
+        const downloadLocation = path.join(tmpDirPath, "openapi.yaml");
+        await fileUtil.downloadFile(requestOptions, downloadLocation);
+        yamlInputFileURI = vscode.Uri.parse(downloadLocation);
+      } catch (e) {
+        console.error(e);
+        vscode.window.showErrorMessage("Failed to download yaml file");
```

Comment:
Maybe this should say `Failed to download yaml file from ${downloadLocation}` to clarify that the extension is downloading the yaml from the url and then generating from that file.